statistics/handle/ddl.go

```diff
@@ -204,6 +219,27 @@ func (h *Handle) updateGlobalStats(tblInfo *model.TableInfo) error {
 	return nil
 }
 
+func (h *Handle) changeGlobalStatsID(from, to int64) (err error) {
+	h.mu.Lock()
+	defer h.mu.Unlock()
```

Comment:
It is modelled after `insertTableStats2KV`. The function will execute 7 internal update statements (change the table ids in each of the stats_* tables, within one transaction, so I expect it to be quick, and during the time it will block accessing non-cached statistics data, and variables.